cdap-docs/admin-manual/source/index.rst

```diff
@@ -96,26 +96,23 @@ installation and its security configuration.
 
 - |operations|_
 
-    .. |logging| replace:: **Logging:**
+    .. |logging| replace:: **Logging and Monitoring:**
     .. _logging: operations/logging.html
 
-    - |logging|_ Covers **CDAP support for logging** through standard SLF4J (Simple Logging Facade for Java) APIs.
+    - |logging|_ CDAP collects **logs and metrics** for all of its internal services and user applications,
+      while CDAP itself can be **monitored through external systems**. Covers **log location**, **logging
```

Comment:
Are you suggesting just have the first sentence, and drop the second? (See the links above for rendered versions of these pages.)